projects/components/src/multi-select/multi-select.component.ts

```diff
@@ -50,6 +50,9 @@ import { MultiSelectJustify } from './multi-select-justify';
             <ht-icon *ngIf="this.icon" [icon]="this.icon" [size]="this.iconSize"> </ht-icon>
             <div *ngIf="!this.isIconOnlyMode()" class="trigger-label-container">
               <ht-label class="trigger-label" [label]="this.triggerLabel"></ht-label>
+              <span *ngIf="this.selectedItemsCount > 1" class="trigger-more-items"
+                >+{{ this.selectedItemsCount - 1 }}</span
```

Comment:
@arjunlalb don't we have a component for this view?

Reply:
There was a ticket hanging around for this. Will dig that up.

Reply:
I will create another PR for it.